### ISSUE TITLE
No module named PyTextNow

### ISSUE THREAD

**Issue by showf68** [leogomezz4t/PyTextNow_API#28]
i installed the module with pip3, and when running the script with python myscript.py, i receive error
    import PyTextNow
ImportError: No module named PyTextNow

**Comment by leogomezz4t** [comment]
Yeah the import goes like this
'import pytextnow' with a lowercase

**Comment by showf68** [comment]
same problem

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
29/06/21,
22:20:32

Le mar. 29 juin 2021 à 22:13, Leo Wu-Gomez ***@***.***> a
écrit :

**Comment by SilverStrings024** [comment]
Can you give us a your code in dpsate.org ? It's difficult to debug someone's code without seeing it.

**Comment by bryanperez43** [comment]
are you using the proper version of python that is linked to your pip3?

i.e.
```
pip3 -V
pip 20.2.2 from /home/bperez/.local/lib/python3.6/site-packages/pip (python 3.6)
```

therefore, you will need to use python3.6 when executing your script

**Comment by showf68** [comment]
ok, now a new problem
/usr/lib/python3/dist-packages/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.26.5) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)

**Comment by bryanperez43** [comment]
```
pip3 install --upgrade --force-reinstall requests
```
should resolve the issue.

**Comment by showf68** [comment]
thanks a lot, it's ok

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
29/06/21,
23:10:19

Le mar. 29 juin 2021 à 23:05, Bryan Perez ***@***.***> a
écrit :

**Comment by bryanperez43** [comment]
I believe PyTextNow version 1.1.7 does not handle the requests dependency with pip install 
this has been fixed for the upcoming 2.0.0 version
